template.yaml

```diff
@@ -698,6 +718,31 @@ Resources:
         - Key: IncludedInBackup
           Value: 'true'
 
+  NvaPersonInfoTable:
```

Comment:
Looks good, but not sure we need more tables.

Reply:
a single table for the whole system is possible, and there are many good reasons for doing so....